src/NumericalAlgorithms/LinearSolver/ExplicitInverse.hpp

```diff
@@ -178,7 +182,10 @@ Convergence::HasConverged ExplicitInverse<LinearSolverRegistrars>::solve(
       // Add a 1 at the unit vector location i
       unit_vector_data = 1.;
       // Invoke the operator on the unit vector
-      linear_operator(make_not_null(&result_buffer), unit_vector);
+      std::apply(linear_operator,
```

Comment:
A protocol doesn't work I think because the linear operator is just an invocable that's intentionally barely constrained, e.g. it can be a lambda function or a struct with a call-operator. It's documented in the `LinearSolver` base class, and I extended the docs a bit in a fixup.